src/components/Uploader/Uploader.tsx

```diff
@@ -39,8 +47,21 @@ export const Uploader = ({ width, height, setValue, getValue, description }: Upl
       setError(!isAllowableSize);
 
       if (isAllowableSize && files[0].type.includes('image')) {
-        setImage(files[0]);
-        setCropPopupVisible(true);
+        if (isPrimaryUiType) {
+          setImage(files[0]);
+          setCropPopupVisible(true);
+        } else {
```

Comment:
you can avoid `else` condition if you add early return in line above

Reply:
done